### ISSUE TITLE
LVR are you still here?

### ISSUE THREAD

**Issue by JLou5641** [PinSimDB/fp-addons-sdk#1]
I LvR,

I'm JLou, creator of FizX for FP, i'm French. Probably it's a bottle in the sea... 
But i wish you are still here or somewhere and see this my message, I would like to ask a question about FPM editor.

I hope you will see it.

Greetings,
JLou

**Comment by sk-1** [comment]
Bonjour JLouYes, I am still there. You found the bottle!I am German, very frequently living in France… ;-)I developed a few items for FP some years ago…like the reverse-engineering of the table checksum…How can I help you?Cheers SK1Von meinem iPhone gesendetAm 23.08.2023 um 13:36 schrieb JLouLoulou ***@***.***>:﻿
I LvR,
I'm JLou, creator of FizX for FP, i'm French. Probably it's a bottle in the sea...
But i wish you are still here or somewhere and see this my message, I would like to ask a question about FPM editor.
I hope you will see it.
Greetings,
JLou

**Comment by JLou5641** [comment]
Ooooh great!

I don't know if you keep an eye on FP evolution we give with Terry.

My question is:

Is it possible to have better resolution of hitbox of Fpm Model?

My wish is to have better resolution for flipper hitbox ... Because they
are pretty low.
It could drasticaly increase flipper accuracy.

And if it possible, is it possible to have just a little update of fpm
editor for that?

I know, ask 2 questions, not only one 🤣🤣.

Greetings,
JLou

**Comment by sk-1** [comment]
Nice to see FP is still ongoing!Honestly, I haven’t tracked latest evolutions… could you provide me a link?About the hitbox: if I remember right, all element graphics are stored in resources, who can be generally edited by one of the libraries or hacking the fp.exe. That’s all I can recall…About the editor: I am afraid this is not possible as I am not the developer and therefore don’t have the source code, which won’t be needed…CheersVon meinem iPhone gesendetAm 23.08.2023 um 15:24 schrieb JLouLoulou ***@***.***>:﻿
Ooooh great!



I don't know if you keep an eye on FP evolution we give with Terry.



My question is:



Is it possible to have better resolution of hitbox of Fpm Model?



My wish is to have better resolution for flipper hitbox ... Because they

are pretty low.

It could drasticaly increase flipper accuracy.



And if it possible, is it possible to have just a little update of fpm

editor for that?



I know, ask 2 questions, not only one 🤣🤣.



Greetings,

JLou

**Comment by JLou5641** [comment]
You can take a look here on Pinball Nirvana and his tutorial:
https://pinballnirvana.com/forums/threads/all-in-one-example-table-fizx-fleep-ball-rolling-pup-ssf-dof-easy-to-add.21983/

and here for one of the Terry's table with Full FizX + SSF + DOF.
https://pinballnirvana.com/forums/resources/jaws-original-bigger-boat-edition-pinevent-v2-fizx-3-0.8020/

If you search for all Terry's table, search with Pinevent keyword ( Star Wars, Junkyard Cats, Blade Runner, Sonic, Jaws, Indiana Jones )...
Also Metal Slug from Anontet.

Be sure you have the lastest BAM update if you want to try.

For model, they are store in library file or directly imported onto table.
![image](https://github.com/PinSimDB/fp-addons-sdk/assets/67042624/0569b52e-174e-4b7f-bd8c-b05f6a75f8ff)
![image](https://github.com/PinSimDB/fp-addons-sdk/assets/67042624/2d3d51cb-bbe7-4433-a3b7-39f8e036e10d)


When i open FP flipper model in fpm editor, you can see the shape has only 10 faces on the tip... I wish to have more face.

**Comment by sk-1** [comment]
Ok will have a look into your project.About the flipper model: I am not a 3d design expert, but the model from the library can be edited.One other idea: have you already asked BAM developer? 100% sure, he can easily overwrite the model with another oneVon meinem iPhone gesendetAm 23.08.2023 um 16:13 schrieb JLouLoulou ***@***.***>:﻿
You can take a look here on Pinball Nirvana and his tutorial:
https://pinballnirvana.com/forums/threads/all-in-one-example-table-fizx-fleep-ball-rolling-pup-ssf-dof-easy-to-add.21983/
and here for one of the Terry's table with Full FizX + SSF + DOF.
https://pinballnirvana.com/forums/resources/jaws-original-bigger-boat-edition-pinevent-v2-fizx-3-0.8020/
If you search for all Terry's table, search with Pinevent keyword ( Star Wars, Junkyard Cats, Blade Runner, Sonic, Jaws, Indiana Jones )...
Also Metal Slug from Anontet.
Be sure you have the lastest BAM update if you want to try.
For model, they are store in library file or directly imported onto table.


When i open FP flipper model in fpm editor, you can see the shape has only 10 faces on the tip... I wish to have more face.

**Comment by JLou5641** [comment]
Yes, model can be edited. I made a lot of model for FizX physics.
But for flipper we can't use use polygon collison, otherwise they don't work anymore with BAM feature. BAM work only with FP hitbox for flipper. When i meant "shape" on my previous answer, i meant "shape of the hitbox in purple".
This is why i ask for hitbox possibilty as it's a FP feature, not BAM feature.

Greetings,

**Comment by JLou5641** [comment]
I forgot, so, who made fpm editor?

**Comment by sk-1** [comment]
Have you already contacted rav (BAM) regarding this?Von meinem iPhone gesendetAm 23.08.2023 um 16:29 schrieb JLouLoulou ***@***.***>:﻿
Yes, model can be edited. I made a lot of model for FizX physics.
But for flipper we can't use use polygon collison, otherwise they don't work anymore with BAM feature. BAM work only with FP hitbox for flipper. When i meant "shape" on my previous answer, i meant "shape of the hitbox in purple".
This is why i ask for hitbox possibilty as it's a FP feature, not BAM feature.
Greetings,

**Comment by JLou5641** [comment]
Yes for sure 😉. I work with him.
He already add lot of BAM new script feature from my wishs and ideas 🤪

**Comment by sk-1** [comment]
Well this is either Christopher Leathley or Martin Antholzner. Martin is responding to requests as far as I know, while Chris is not…Von meinem iPhone gesendetAm 23.08.2023 um 16:34 schrieb JLouLoulou ***@***.***>:﻿
I forgot, so, who made fpm editor?

**Comment by JLou5641** [comment]
Thanks! 😉

Where i can contact Martin?

**Comment by beta-tester** [comment]
huuh... ?!
i see activity here !

remembers me to the good "old" days where GoPinball and PinSimDB websites were active, clean from spam and online...
all gone... where is all the good stuff gone to?

i collision model of a flipper must be able to customize...
i thing there was a table with banana flippers or other non default flipper models on FP tables.
or is my memory gone too... ?

also my very old FPM/FPL/FPT importer for the very old Blender 2.79 had an option to import the collision model into Blender.

here some links in case some one is still locking for;
- [Doc: FPM-/FPL-/FPT-Importer](https://archive.blender.org/wiki/index.php/Extensions:2.6/Py/Scripts/Import-Export/FuturePinball_FPx/)
- [Doc: MS3D Imprter/Exporter](https://archive.blender.org/wiki/index.php/Extensions:2.6/Py/Scripts/Import-Export/MilkShape3D_MS3D/)
- [Blender 2.79-e045fe53f1b0 with MS3D-Importer/Exporter and FPM-/FPL-/FPT-Importer add-ons included](https://download.blender.org/release/Blender2.79/latest/)
- [some videos about the add-ons](https://vimeo.com/user5569931)

**Comment by JLou5641** [comment]
Hello,

Yes, FP has a big step this year 😉
Look for FizX 😉

Yes collision model are customisable, but not their number of polygone.
From my banana flipper, from my memory, it's only visual, not really physics. Collision model is still like a regular flipper.

Thanks for the docs,
i think nobody have those docs!

**Comment by JLou5641** [comment]
"also my very old FPM/FPL/FPT importer for the very old Blender 2.79 had an option to import the collision model into Blender."


Are you sure?

**Comment by JLou5641** [comment]
> huuh... ?! i see activity here !
> 
> remembers me to the good "old" days where GoPinball and PinSimDB websites were active, clean from spam and online... all gone... where is all the good stuff gone to?
> 
> i collision model of a flipper must be able to customize... i thing there was a table with banana flippers or other non default flipper models on FP tables. or is my memory gone too... ?
> 
> also my very old FPM/FPL/FPT importer for the very old Blender 2.79 had an option to import the collision model into Blender.
> 
> here some links in case some one is still locking for;
> 
> * [Doc: FPM-/FPL-/FPT-Importer](https://archive.blender.org/wiki/index.php/Extensions:2.6/Py/Scripts/Import-Export/FuturePinball_FPx/)
> * [Doc: MS3D Imprter/Exporter](https://archive.blender.org/wiki/index.php/Extensions:2.6/Py/Scripts/Import-Export/MilkShape3D_MS3D/)
> * [Blender 2.79-e045fe53f1b0 with MS3D-Importer/Exporter and FPM-/FPL-/FPT-Importer add-ons included](https://download.blender.org/release/Blender2.79/latest/)
> * [some videos about the add-ons](https://vimeo.com/user5569931)

I check an eye on "import collision model"... as I thought, it's not possible.
![image](https://github.com/PinSimDB/fp-addons-sdk/assets/67042624/4c257c54-8c41-439b-9048-f9439347b963)

"Not implementing yet". Even if would be possible, we have nothing to import in FP custom collision model in Fpm Editor.

Greetings,
JLou

**Comment by beta-tester** [comment]
oops... it isn't implemented yet?!

sorry, i could not remember me to this. i just watched the buttons on the picture.
but it also means that it is possible, but i was too lazy to implement it.
let me search in the code for a hint...

**Comment by JLou5641** [comment]
> oops... it isn't implemented yet?!
> 
> sorry, i could not remember me to this. i just watched the buttons on the picture. but it also means that it is possible, but i was too lazy to implement it. let me search in the code for a hint...

🤣🤣. No problem 😉

By the way, is it possible to have this for newer Blender version?
We mainly use newer version a Blender, so each time we import to blender 2.79, then blender 2.79 to newer... then to milkashape or blender 2.79 again to export to FP with Fpm editor or BAM Model.

We will apreciate a lot if it possible as it could simplify a lot our workflow.

Greetings
JLou

**Comment by beta-tester** [comment]
> let me search in the code for a hint...

no, sorry, i was wrong. the "collision data" / "collision per polygon" / "shape collision" is not what i thought yesterday.


> By the way, is it possible to have this for newer Blender version?

unfortunately meanwhile many many things changed since blender 2.79.
i tried to port the ms3d add-on from 2.79 to 3.x, but i lost interest in it very quickly.
because the ms3d add-on not finished porting, so FPx importer was never touched by me anymore.

the last contact was here:
[Import-Export MS3D - addon ported from blender 2.79 to blender 3.2 #98826](https://projects.blender.org/blender/blender-addons/issues/98826)

there is a guy porting the add-on possibly:
[rollin](https://projects.blender.org/rollin) commented: ... [I'm converting the plugin to blender 2.8 / 3.X.](https://projects.blender.org/blender/blender-addons/issues/29404#issuecomment-68485)

**Comment by JLou5641** [comment]
Yep 😉.. This is what I thought about collision data.

No problem for the update. We will continue like that. Thanks a lot.

Greetings,
JLou
